Goobi/src/de/sub/goobi/metadaten/Paginator.java

```diff
@@ -99,9 +99,20 @@ private void assertSelectionIsNotNull() {
 		}
 	}
 
+	/**
+	 * Throws NumberFormatException if `paginationStartValue` isn’t a valid
+	 * number of the type specified by `paginationType`.
+	 * 
+	 * @throws NumberFormatException
+	 *             if `paginationStartValue` isn’t valid
+	 */
 	private void assertValidPaginationStartValue() {
 		// arabic numbers
 		if (paginationType == Paginator.Type.ARABIC) {
+			/* coverity[USELESS_CALL]
+			 * Integer.parseInt() throws NumberFormatException if
+			 * paginationStartValue cannot be parsed to an `int`, and this is
+			 * what we want here. */
```

Comment:
I don’t find it even too ugly, I first thought about (and tried to implement correctly) manually checking. [Here is an example, but it does not take into account numeric overflow and does not iterate over code points, btw.](http://stackoverflow.com/a/237204/1503237)—and it is a lot of code. Second point is that `RomanNumeral.setValue(String)` (l. 121) behaves the same and also throws NumberFormatException if the input is incorrect. So both methods behave the same. It’s still a bit smelly, yes.